src/p5.dance.js

```diff
@@ -183,6 +183,17 @@ module.exports = class DanceParty {
     this.allSpritesLoaded = true;
   }
 
+  setUserBlocks(userBlocks) {
```

Comment:
Minor, but would it be better to make `userBlocks` a require parameter to `play()`? I worry that we might get into an inconsistent state if we call `play()` but forget to call `setUserBlocks`, but this way we're always passing in the latest set of blocks when running the program.

Reply:
Omg - I just love this suggestions. Just updated - thanks!